### PR TITLE
feat(proteus): Add WASMUtil

### PR DESCRIPTION
**Pull request by ffflorian**
## Pull Request Checklist

- [x] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/proteus/spec/util/WasmUtilSpec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+const Proteus = require('@wireapp/proteus');
+
+describe('WASMUtil', () => {
+  it('Uses WebAssembly', async () => {
```
Comment (bennyn) [inline packages/proteus/spec/util/WasmUtilSpec.js]: We need a `describe` block for `isUsingWASM` before using an `it` block.

Comment (ffflorian) [reply packages/proteus/spec/util/WasmUtilSpec.js]: Whoops, thanks!

File: packages/proteus/spec/util/WasmUtilSpec.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+const Proteus = require('@wireapp/proteus');
+
+describe('WASMUtil', () => {
+  describe('isUsingWASM', () => {
+    it('Uses WebAssembly', async () => {
```
Comment (bennyn) [inline packages/proteus/spec/util/WasmUtilSpec.js]:
```suggestion
    it('reports if the target environment supports WebAssembly', async () => {
```

File: packages/proteus/spec/util/WasmUtilSpec.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      // To test the other way round, we would need to start Node.js
+      // with the flag `--noexpose_wasm`.
+      const result = await Proteus.util.WASMUtil.isUsingWASM();
```
Comment (bennyn) [inline packages/proteus/spec/util/WasmUtilSpec.js]:
```suggestion
      const isUsingWASM = await Proteus.util.WASMUtil.isUsingWASM();
```

File: packages/proteus/spec/util/WasmUtilSpec.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      expect(result).toBe(true);
```
Comment (bennyn) [inline packages/proteus/spec/util/WasmUtilSpec.js]:
```suggestion
      expect(isUsingWASM).toBe(true);
```

**Pull request merged**